source/clear-linux/concepts/swupd-about.rst

```diff
@@ -129,6 +129,13 @@ system and its updates as the basis. Using this tool, system administrators
 can focus on the custom pieces their deployments require while staying on
 a controlled update stream.
 
+Next steps
+==========
+
+To put this concept into practice, we have the following step-by-step
```

Comment:
Why are we putting the concept into action in the concept section. This makes no sense. If the intent is to cross-reference the guide, there is no need for a whole new section called next steps. Just a single sentence along the lines of: To learn how to run an update of your system, visit our :ref:`update`.